FinanceServicesApi/V1/Boundary/Request/Enums/AssetType.cs

```diff
@@ -0,0 +1,9 @@
+namespace FinanceServicesApi.V1.Boundary.Request.Enums
+{
+    public enum AssetType
+    {
+        Property,
```

Comment:
We don't need Asset Type as Property, as the request will pass asset type as Dwelling for Property.

Reply:
Done